### PR TITLE
Fixing references from previous PR

### PR DESCRIPTION
**Pull request by rockdrigofc**
Adding fixed references and comments from last commit. Still not persistant phone dir

### CODE REVIEW

File: exercises/phone-directory-cpp/CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.10)
 
 set (CMAKE_BUILD_TYPE Debug)
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-set (CMAKE_CXX_COMPILER "D:\\mingw\\bin\\mingw32-g++.exe")
```
Comment (NeoraptX) [inline exercises/phone-directory-cpp/CMakeLists.txt:5]: It's better to have this set in runtime, path is relative to the user

File: exercises/phone-directory-cpp/inc/contactdata.h [hunk 2]
```diff
@@ -1,7 +1,8 @@
 
-//#include <iostream>
-#include <string>
-class ContactData {
+#include <iostream>
+//#include <string>
```
Comment (NeoraptX) [inline exercises/phone-directory-cpp/inc/contactdata.h]: why this comment?

File: exercises/phone-directory-cpp/src/singleton.cpp [hunk 3]
```diff
@@ -1,18 +1,19 @@
-#include "../header/singleton.h"
+#include "../inc/singleton.h"
```
Comment (NeoraptX) [inline exercises/phone-directory-cpp/src/singleton.cpp]: If includepath is specified in CMake then it should be possible to add this include as:
#include "singleton.h"

**Review by NeoraptX — CHANGES_REQUESTED** [review]

**Review by JuanCodeLab — COMMENTED** [review]
No new comments

File: exercises/phone-directory-cpp/inc/contactdata.h (hunk 2 above)
Comment (rockdrigofc) [reply exercises/phone-directory-cpp/inc/contactdata.h]: Not used, removed in submitted commit

File: exercises/phone-directory-cpp/src/singleton.cpp (hunk 3 above)
Comment (rockdrigofc) [reply exercises/phone-directory-cpp/src/singleton.cpp]: Fixed in last commit

File: exercises/phone-directory-cpp/CMakeLists.txt (hunk 1 above)
Comment (rockdrigofc) [reply exercises/phone-directory-cpp/CMakeLists.txt:5]: Fixed

**Review by NeoraptX — APPROVED** [review]

**Review by JuanCodeLab — APPROVED** [review]

**Pull request merged**